11/umbraco-cms/fundamentals/backoffice/settings.md

```diff
@@ -0,0 +1,83 @@
+---
+description: A guide displaying the options available in the Settings section in Umbraco Backoffice.
+---
+
+# Backoffice Settings Section
+
+The **Settings** section of the Umbraco backoffice has its own set of default dashboards:
+
+<details>
+
+<summary>Welcome</summary>
+
+The Welcome dashboard is part of the Settings section. It is displayed next to the Settings tree and consists of a blank view that can be used to build any custom dashboard. The Welcome dashboard can be customized to display information as per your needs. It is possible to extend the dashboard and add them to either an existing section or a custom one.
+
+For more information about creating custom dashoards, see the [Dashboards](../extending/dashboards.md) article.
+
+</details>
+
+<details>
+
+<summary>Examine Management</summary>
+
+The Examine Management dashboard provides an overview of the Examine functionality available directly within the Umbraco backoffice. The Umbraco backoffice allows you to view details about your Examine indexes and searchers - all in one place. You can see which fields are being indexed and rebuild the indexes if there's a problem. You can also test keywords to see what results will be returned.
+
+For more information about Examine Management, see the [Examine Management](../reference/searching/examine/examine-management.md) article.
+
+</details>
+
+<details>
+
+<summary>Published Status</summary>
+
+The Published Status dashboard displays the status of your site in the Published Cache Status section alongwith the Content and Media nodes value. The Caches section provides three options: Memory Cache, Database Cache, and Internals (NuCache).
+
+- Memory Cache - Reloads the in-memory cache by entirely reloading it from the database cache. Use it when you think that the memory cache has not been properly refreshed.
+
+- Database Cache - Rebuilds the database cache that is the content of the `cmsContentNu` table. Use it when reloading the Memory Cache is not enough and you think that the database cache has not been properly generated.
+
+- Internals - Lets you trigger a NuCache snapshots collection.
+
+</details>
+
+<details>
+
+<summary>Models Builder</summary>
+
+Models builder is a tool that can generate a complete set of strongly-typed published content models for Umbraco. Models are available in both controllers and views.
+
+For more information about Models Builder, see the [Models Builder](../reference/templating/modelsbuilder/README.md) article.
+
+</details>
+
+<details>
+
+<summary>Health Check</summary>
+
+Health Checks are used to determine the status of your Umbraco project. It is a handy list of checks to see if your Umbraco installation is configured according to best practices. It's possible to add your custom-built health checks.
+
+For more information about Health Checks, see the [Health Check](health-check/README.md) article.
```

Comment:
Added an "s" to indicate there are many articles. Feel free to remove if it doesn't make sense 😅 

```suggestion
For more information about Health Checks, see the [Health Check](health-check/README.md) articles.
```